elyra/pipeline/processor_airflow.py

```diff
@@ -45,6 +49,29 @@ class AirflowPipelineProcessor(RuntimePipelineProcess):
     def type(self):
         return self._type
 
+    @property
+    def component_registry(self) -> str:
```

Comment:
How does this property differ from `Component.registry_location` (which is the correct name IMO).

I think it might be better if this property was object-valued and held the instance of the `ComponentRegistry` that is instantiated during construction below.

Reply:
Renamed the different incarnation of the property to all use the same name. We will try to add something to abstract the registry catalog and the additional changes will come with that pr.